### PR TITLE
[EZP-28180] Adding function "create" to simplify the creation of a ezsitedata entry

### PR DESCRIPTION
**Pull request by pkamps**
Most changes are due to code reformatting. I added a new function "create" to simplify the creation process.
The function has a new unit test.

### CODE REVIEW

**Comment by peterkeung** [comment]
+1

We've been using the ezsite_data table more as convenient and simple persistent storage.

**Comment by brookinsconsulting** [comment]
+1

**Comment by andrerom** [comment]
Issue?

**Comment by pkamps** [comment]
https://jira.ez.no/browse/EZP-28180

File: kernel/classes/ezsitedata.php [hunk 1]
```diff
@@ -8,38 +8,64 @@
...
-                                                                             'default'  => null,
-                                                                             'required' => true ),
-
-                                            ),
-
-                      'keys'                 => array( 'name' ),
-                      'class_name'           => 'eZSiteData',
-                      'name'                 => 'ezsite_data',
-                      'function_attributes'  => array()
+        return array(
```
Comment (andrerom) [inline kernel/classes/ezsitedata.php]: any actual need to change this? seems to be formatting only.

File: kernel/classes/ezsitedata.php [hunk 2]
```diff
@@ -8,38 +8,64 @@
...
+    {
+        return new eZSiteData( array(
+            'name' => $name,
+            'value' => $value
+        ) );
+    }
+
     /**
      * Fetches a site data by name
+	 *
```
Comment (andrerom) [inline kernel/classes/ezsitedata.php]: wrong indentation

**Comment by pkamps** [comment]
Feedback implemented.

**Pull request merged**